### ISSUE TITLE
Order tags alphabetically

### ISSUE THREAD

**Issue by lucasgranberg** [sonata-project/SonataNewsBundle#246]
I can't seem to find a way to order the tags in the select2 dropdown. I would like to use doctrines order-by mappin, like this.
```xml
<order-by>
    <order-by-field name="name" direction="ASC" />
</order-by>
```
I tried using the association override in my easyextended newsbundle like this:
```xml
<association-overrides>
    <association-override name="tags">
        <order-by>
            <order-by-field name="name" direction="ASC" />
       </order-by>
   </association-override>
</association-overrides>
```
but it doesnt work, I get the following error.
```
Invalid field override named 'tags' for class 'Application\Sonata\NewsBundle\Entity\Post
```
I suspect it is becouse of the mapping being done in the [dependencyinjector](https://github.com/sonata-project/SonataNewsBundle/blob/master/DependencyInjection/SonataNewsExtension.php#L219) but Im not sure in what order the different configurations are used.

Anyway I suppose ordering by name should be the default, maybe it could be done from sonata admin or via javascript but it seems logical to me that ordering by name would be the default.

**Comment by sagikazarmark** [comment]
:+1:

**Comment by OskarStark** [comment]
:+1: , but I'm not so familar with the NewsBundle

anyone willing to provide a PR?

**Comment by sagikazarmark** [comment]
Actually I would solve this issue at the Admin level. I wouldn't add this ordering right into the Entity. AFAIK you can create custom queries in the admin form mapper (even for relations). Tags should be sorted there in a custom query.

**Comment by OskarStark** [comment]
I would solve this on Entity-Level because you want the same ordering in the frontend, right?

**Comment by sagikazarmark** [comment]
Not necessarily. For example you could order tags by the highest number of posts under a specific tag. Or most popular tag based on views. Personally I think that the less logic in entities, the better it is.
